content/registration_pages/confirmation.en.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Confirmation
+date: 2024-02-22T05:58:03.337Z
+description: Confirmation
+---
```

Comment:
**Annotation**:
Content in English/Korean can be added to these pages. The Javascript can redirect to `/registration_pages/confirmation` but I was not able to get it to preserve `/ko` or not.  Might have to do some logic in the react component.